site/cypress/e2e/gradeables/gradeable_results.spec.js

```diff
@@ -0,0 +1,51 @@
+
+
+describe('Testing functionality of Autograder Results', () => {
+    beforeEach(() => {
+        cy.visit(['sample','gradeable','closed_team_homework']);
+        cy.login('student');
+
+    });
+
+    it('Should display confetti', () => {
+        cy.get('#confetti_canvas[style="display: block;"]').should('not.exist');
+        cy.get('div[onclick="addConfetti();"]').click();
+        cy.get('#confetti_canvas[style="display: block;"]').should('exist');
```

Comment:
These need to be targeted in a more specific way.  Targeting based on a JS function name or an inline CSS style is fragile.  It would be more robust to target these elements based on an ID or class, or via a data tag similar to what Cameron did in #9503.

Reply:
done